src/app/user/shared/comment/service/comment.service.ts

```diff
@@ -0,0 +1,61 @@
+import {Injectable} from '@angular/core';
+import {CommentManagerService} from '../manager/comment-manager.service';
+import {CreateCommentRequest} from '../request/create-comment-request';
+import {UpdateCommentRequest} from '../request/update-comment-request';
+import {RouteToAPIService} from '../helper/route-to-api.service';
+import {EMPTY, Observable, Subject} from 'rxjs';
+import {CommentObject} from '../entity/comment-object';
+import {catchError} from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CommentService {
```

Comment:
same as before ,
one service for each type

Reply:
Since the Code is very duplicated, i tried to extend base-comment and add entity specific methods, is that OK?!!